examples/pytorch/language-modeling/run_clm.py

```diff
@@ -401,6 +404,7 @@ def group_texts(examples):
         batched=True,
         num_proc=data_args.preprocessing_num_workers,
         load_from_cache_file=not data_args.overwrite_cache,
+        desc="Running tokenizer on concatenated texts",  # not sure if it's right
```

Comment:
```suggestion
        desc=f"Grouping texts in chunks of {max_seq_length}"
```

Reply:
There's no `max_seq_length` variable here :/

Reply:
Sorry, it's `block_size` in this example.